src/main/java/net/dries007/tfc/api/capability/player/CapabilityPlayer.java

```diff
@@ -30,28 +30,28 @@
 
 import static net.dries007.tfc.api.util.TFCConstants.MOD_ID;
 
-public final class CapabilityFood
+public final class CapabilityPlayer
 {
     public static final float MIN_PLAYER_NUTRIENTS = 0f;
     public static final float MAX_PLAYER_NUTRIENTS = 100f;
     public static final int DEFAULT_ROT_TICKS = CalendarTFC.TICKS_IN_DAY * 15;
 
     @CapabilityInject(IFood.class)
     public static final Capability<IFood> CAPABILITY_NUTRIENTS = Helpers.getNull();
-    @CapabilityInject(IPlayerNutrients.class)
-    public static final Capability<IPlayerNutrients> CAPABILITY_PLAYER_NUTRIENTS = Helpers.getNull();
+    @CapabilityInject(IPlayerData.class)
+    public static final Capability<IPlayerData> CAPABILITY_PLAYER_DATA = Helpers.getNull();
 
     private static final ResourceLocation KEY = new ResourceLocation(MOD_ID, "nutrients");
-    private static final ResourceLocation PLAYER_KEY = new ResourceLocation(MOD_ID, "player_nutrients");
+    private static final ResourceLocation PLAYER_KEY = new ResourceLocation(MOD_ID, "player_data");
 
     private static List<Supplier<PotionEffect>> rottenFoodEffects = null;
 
     public static void preInit()
     {
-        // Item nutrient capability
+        // Item player capability
         CapabilityManager.INSTANCE.register(IFood.class, new DumbStorage<>(), FoodHandler::new);
-        // Player nutrient capability
-        CapabilityManager.INSTANCE.register(IPlayerNutrients.class, new DumbStorage<>(), PlayerNutrientsHandler::new);
+        // Player player capability
```

Comment:
You need to fix a bunch of these errors all over - take care when doing an refactor in Intellij to not grab comments that aren't part of what you are trying to rename.

Reply:
Slap me in the face, I missed another one :X